HW02_A_Graph-Fails.Rmd

```diff
@@ -1,87 +1,153 @@
+```
+
+### Graph Fail 2
+Why aren't the points blue? It is making me blue that the points in the graph aren't blue :`(
+```{r}
+
+#ORIGINAL CODE WITH ERROR:
+ggplot(data = mpg) + 
+  geom_point(mapping = aes(x = displ, y = hwy, color = "blue"))
+
+#EDITED CODE:
+ggplot(data = mpg, aes(x = displ, y = hwy)) + 
+  geom_point(color = "blue")
+
+```
+
+
+### Graph Fail 3
+Two mistakes in this graph. First, I wanted to make the the points slightly bolder, but changing the alpha to 2 does nothing. What does alpha do and what does setting it to 2 do? What could be done instead if I want the points slightly bigger? 
+
+ANSWER: 
+Changing the alpha value changes the transparency of the points:
+  alpha = 1 : opaque 
+  alpha = 0 : transparent
+  
+  i am not sure what setting alpha = 2 does.  i can not see any changes when i generate the plot
+  
+  you could use size to make the points larger
+
+Second, I wanted to move the legend on top of the graph since there aren't any points there, putting it at approximately the point/ordered pair (5, 40). How do you actually do this? Also, how do you remove the legend title ("class")? Finally, how would you remove the plot legend completely?
+
+ANSWER:
+  c(1,1) - places the legend in the upper right hand corner, start there and then adjust the numbers untl in the right location
+  add legend.title = element_blank() to remove the title
+  change legend.position = "none" to remove the plot legend entirely
+```{r}
+#ORIGINAL CODE WITH ERROR
+mpg %>% 
+ggplot() + 
+  geom_point(mapping = aes(x = displ, y = hwy, color = class), alpha = 2) + 
+  theme(legend.direction = "horizontal") + 
+  theme(legend.position = c(5, 40))
+
+#EDITED CODE
+mpg %>% 
+ggplot() + 
+  geom_point(mapping = aes(x = displ, y = hwy, color = class), size = 3) + 
+  theme(legend.direction = "horizontal") + 
+  theme(legend.position = c(0.6,0.85)) +
+  theme(legend.title = element_blank())
+  
+#to remove the legend completely
+mpg %>% 
+ggplot() + 
+  geom_point(mapping = aes(x = displ, y = hwy, color = class), size = 3) + 
+  theme(legend.direction = "horizontal") + 
+  theme(legend.position = "none")
+
+```
+
+
+### Graph Fail 4
+I wanted just one smoothing line. Just one line, to show the general relationship here. But that's not happening. Instead I'm getting 3 lines, why and fix it please?
+
+ANSWER:
+so when i removed color = dry the three different color distincts disappear and there is one smooth line going across the entire dataset.  i am not sure if there is a way to have one smooth line going across the data with it being grouped by the three different categories?
+
+```{r}
+#ORIGINAL CODE
+mpg %>% 
+ggplot(mapping = aes(x = displ, y = hwy, color = drv)) + 
+  geom_point() + 
+  geom_smooth(se = F) #se = F makes it so it won't show the error in the line of fit
+
+#EDITED CODE
+mpg %>% 
+ggplot(mapping = aes(x = displ, y = hwy)) + 
+  geom_point() + 
+  geom_smooth(se = F) #se = F makes it so it won't show the error in the line of fit
```

Comment:
Hi Ariana, 

I was facing the same problem you had and I found this helpful post on stack overflow. You can refer to this link https://stackoverflow.com/questions/47883446/plot-a-single-geom-smooth-curve-instead-of-multiple or my codes for more details but essentially, there are two ways to solve this issue. 

1) Override the color aesthetic in geom_smooth layer
2) Move the color aesthetic to only layers that need it